fragdenstaat_de/fds_donation/templates/admin/fds_donation/donor/donor_total_amount_per_year_filter.html

```diff
@@ -0,0 +1,32 @@
+{% load i18n static %}
+<!-- Custom filter for donor total amount per year -->
+<details data-filter-title="DonorTotalAmountPerYear" open="">
+    <summary>{{ spec.title }}</summary>
+    <ul>
+        <li>
+            <form action="" method="get">
+                <p>
+                    <label for="id_amount">{% trans "Amount from - to:" %}</label>
+                    <input type="text"
+                           id="id_amount"
+                           value="{{ spec.amount_str|escape }}"
+                           placeholder="1000-">
+                </p>
+                <p>
+                    <label for="id_year">{% trans "In year:" %}</label>
+                    <input type="text"
```

Comment:
This should probably `type="number"` and use [inputmode="numeric"](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/inputmode).

Ideally, there's no placeholder and the initial value is always the last year and it becomes an easy replacement for the amount_last_year filter. Together with the spinbutton of `type="number"` it becomes easy to cycle through recent years – but only if a year is prefilled.